src/Kdyby/Monolog/Diagnostics/MonologAdapter.php

```diff
@@ -46,6 +51,15 @@ public function __construct(Monolog\Logger $monolog)
 
 		$this->directory = &Debugger::$logDirectory;
 		$this->email = &Debugger::$email;
+
+		$this->priorityMap = array(
```

Comment:
If it were somehow dynamic, it would be better in construtor, but here it's completely static.